### PR TITLE
Feature: Github action to run tests on Linux

### LINKED ISSUE #62: setup CI for automated tests on Linux

### PR DESCRIPTION
**Pull request by sudiptob2**
closes #62

### CODE REVIEW

File: .github/workflows/test-linux.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Test on Linux
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```
Comment (0hsn) [inline .github/workflows/test-linux.yml:7]: should support [Ubuntu 20.04.5 LTS (Focal Fossa)](https://releases.ubuntu.com/focal/)

File: .github/workflows/test-linux.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
```
Comment (0hsn) [inline .github/workflows/test-linux.yml:15]: please confirm `> 3.10.5`

**Review by 0hsn — COMMENTED** [review]
`.github/workflows/test-linux.yml` -> `.github/workflows/linux-run-tests.yml`

File: .github/workflows/test-linux.yml (hunk 1 above)
Comment (sudiptob2) [reply .github/workflows/test-linux.yml:7]: upto [ubuntu-22.04.1  ](https://github.com/actions/runner-images) is available now

File: .github/workflows/linux-run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10.5'
+          architecture: 'x64'
+
+      - name: Python version
+        run: python -c "import sys; print(sys.version)"
+
+      - name: Install pipenv
+        run: python -m pip install --upgrade pipenv wheel
```
Comment (0hsn) [inline .github/workflows/linux-run-tests.yml:22]: why do we need wheel for?

**Review by 0hsn — APPROVED** [review]

File: .github/workflows/linux-run-tests.yml (hunk 3 above)
Comment (sudiptob2) [reply .github/workflows/linux-run-tests.yml:22]: We don't need it, will update

File: .github/workflows/test-linux.yml (hunk 1 above)
Comment (0hsn) [reply .github/workflows/test-linux.yml]: should work!

**Pull request merged**